dolphinscheduler-task-plugin/dolphinscheduler-task-seatunnel/src/main/java/org/apache/dolphinscheduler/plugin/task/seatunnel/SeatunnelTask.java

```diff
@@ -115,7 +115,8 @@ private String buildCommand() throws Exception {
         args.addAll(buildOptions());
 
         String command = String.join(" ", args);
-        logger.info("SeaTunnel Flink task command: {}", command);
```

Comment:
This log output is wrong, it should be `logger.info("SeaTunnel task command: {}", command);`

Reply:
ok